app/controllers/search_controller.rb

```diff
@@ -0,0 +1,16 @@
+class SearchController < ApplicationController
+  def search
+    
+  end
+
+  def suggest
+    if params['query'].to_s != ''
+      param = "%#{params['query']}%".downcase
+      @results =  User.where("lower(usr_lastname || ' ' || usr_firstname) LIKE ? OR lower(usr_nickname) LIKE ?\
+       OR lower(usr_email) LIKE ? OR usr_cell_phone LIKE ?", param, param, param, param).limit(10)
+
+      render partial: 'suggest_partial'
```

Comment:
Ezzel a partial renderrel tobb problema is van:

1. Szerintem a `_partial` a neveben teljesen felesleges. A file amit renderelni fog ugyis a `_` karakterrel kezdodik, szoval elkulonul a tobbitol
2. Ha partialt rendelsz akkor a instance variable szerintem veszelyes. Nem egy `@result = foo` csinalnek hanem inkabb explicit atadnam at, hogy latszon a partialnek van valamilyen fuggosege. Szoval: `render partial: 'suggest', locals: { result: result }`
3. Railsben a JS kezeles altalaban ugy szokott lenni, hogy a valaszban olyan js-t kuldesz le amit a kliens automatikusan megfuttat es a valaszban kapott js kod valtoztatja a domot. http://railscasts.com/episodes/136-jquery (kicsit regi de azert relevans). Itt persze nem rossz a mostani megoldas sem, csak akkor kicsit a framework ellen kell menni.